app/errors.py

```
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    def __init__(self, name: str = "NotVaccinatedError") -> None:
+        super().__init__(name)
+
+
+class OutdatedVaccineError(VaccineError):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+
+
+class NotWearingMaskError(Exception):
+    def __init__(self, name: str) -> None:
```

Let's also provide a descriptive default message here, for example: `'Visitor must wear a mask.'`